AES.Entities/Tables/Application.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace AES.Entities.Tables
+{
+    public class Application
+    {
+        [Key]
+        [Required]
+        public int ID { get; set; }
+
+        // Which job the application is for
```

Comment:
Instead of a single line comment, type three slashes, which will create a <summary> section. Anything between the start and close tags will be visible when somebody uses the properties. Repeat for similar property explanations.